doc/glm/first_level_model.rst

```diff
@@ -0,0 +1,219 @@
+.. _first_level_model:
+
+=======================================================
+First level models
+=======================================================
+
+.. topic:: **Page summary**
+
+  First level models are, in essence, linear regression models run at the level of a single
+  session or single subject. The model is applied on a voxel-wise basis, either on the whole
+  brain or within a region of interest. The timecourse of each voxel is regressed against a
+  predicted BOLD response created by convolving the haemodynamic response function (HRF) with
+  a set of predictors defined within the design matrix.
+
+
+.. contents:: **Contents**
+    :local:
+    :depth: 1
+
+
+HRF models
+=====================================================
+
+Nilearn offers a few different HRF models including the commonly used double-gamma SPM model
+('spm') and the model shape proposed by G. Glover ('glover'), both allowing the option of adding
+time and dispersion derivatives. The addition of these derivatives allows to better model any
+uncertainty in timing information. In addition, an FIR (finite impulse response, 'fir') model
+of the HRF is also available.
+
+In order to visualize the predicted regressor prior to plugging it into the linear model, use
+the function :func:`nilearn.stats.first_level_model.compute_regressor`, or explore the HRF plotting
+example: :ref:`sphx_glr_auto_examples_04_glm_first_level_models_plot_hrf.py`.
+
+
+Design matrix: event-based and time series-based
+=====================================================
+
+Event-based
+-----------------------------------------------------
+
+To create an event-based design matrix, information about the trial type, onset time and
+duration of the events in the experiment are necessary. This can be provided by the user, or
+be part of the dataset if using one of the nilearn dataset fetcher functions like
+:func:`nilearn.datasets.fetch_spm_multimodal_fmri`,
+:func:`nilearn.datasets.fetch_language_localizer_demo_dataset`, etc.
+Using a nilearn fetcher function, e.g. :func:`nilearn.datasets.fetch_spm_multimodal_fmri`,
+first download the data (for one subject)::
+
```

Comment:
Make it explicit that the adjunction of an events.tsv file is part of the BIDS standard